daemon/c_vol.c

```diff
@@ -603,18 +582,23 @@ c_vol_mount_image(c_vol_t *vol, const char *root, const mount_entry_t *mntent)
 	case MOUNT_TYPE_SHARED_RW:
 	case MOUNT_TYPE_OVERLAY_RW:
 		overlay = true;
+		shiftids = true;
 		break;
 	case MOUNT_TYPE_DEVICE_RW:
 	case MOUNT_TYPE_EMPTY:
+		shiftids = true;
 		break; // stick to defaults
 	case MOUNT_TYPE_BIND_FILE:
 		mountflags |= MS_RDONLY; // Fallthrough
 	case MOUNT_TYPE_BIND_FILE_RW:
+		if (container_has_userns(vol->container)) // skip
+			goto final;
```

Comment:
nop we just skip silently otherwise mount would return with an error and container wont start. This is to disable the bind-file feature if userns is enable. To enable this again more work needs to be done in a separate patch.